KS.Fiks.IO.Client/Configuration/FiksIOConfiguration.cs

```diff
@@ -81,7 +84,8 @@ public class FiksIOConfiguration
             Guid fiksKontoId,
             string privatNokkel,
             string issuer,
-            X509Certificate2 sertifikat,
+            X509Certificate2 maskinportenSertifikat,
+            X509Certificate2 asiceSertifikat,
```

Comment:
her

Reply:
her mener jeg det ikke er missvisende? Her er den på linje 87 for maskinporten og den på linje 88 for asice. Man skal kunne ha 2 forskjellige?